pom.xml

```diff
@@ -27,13 +27,13 @@
   <parent>
     <groupId>com.amazon.opendistroforelasticsearch</groupId>
     <artifactId>opendistro_security_parent</artifactId>
-    <version>0.9.0.4</version>
+    <version>0.9.0.2</version>
```

Comment:
Why are we changing version here ?

Reply:
The latest backend security plugin for ES6.7 is v0.9.0.2.